File: round10/git_exercise/Temperature.java
import java.util.Scanner;

public class Temperature {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        try {
            System.out.print("Input temperature: ");
            double inputTemp = Double.parseDouble(scanner.nextLine());
            System.out.printf("%.2f C = %.2f F\n",
                    inputTemp, tTransform(inputTemp, 'c'));
            System.out.printf("%.2f F = %.2f C\n",
                    inputTemp, tTransform(inputTemp, 'f'));
        } catch (NumberFormatException e) {
            System.out.println("Not valid input");
            System.exit(1);
        }
    }

    public static double tTransform(double temp, char type) {
        if (type == 'c') {
            return temp * 9/15 + 32;
        }
        if (type == 'f') {
            return (temp - 32) * 15/9;
        }
        return Double.NaN;
    }
}

File: round10/git_exercise/temperature.py

def main():

    input_ask = string(input("Select transform direction (CF) or (FC): "))

    if input_ask == "CF":
        try:
            input_temp = float(input("Input temperature: "))
        except:
            print("Not valid input")
            exit()

    print("{} C = {} F".format(input_temp, t_transform(input_temp, 'c')))
        
    elif input_ask == "FC":
        try:
            input_temp = float(input("Input temperature: "))
        except:
            print("Not valid input")
            exit()

    print("{} F = {} C".format(input_temp, t_transform(input_temp, 'f')))

    else:
        print("Not valid input")
        exit()

def t_transform(temp, type):

    if type == 'c':
        return float(temp * 9/5 + 32)
    elif type == 'f':
        return float((temp - 32) * 5/9)

    return None

if __name__ == "__main__":
    main()
